### DISCUSSION TITLE
Conversion stops after 1/3

### DISCUSSION THREAD

**Discussion by HenriRT** [gildas-lormeau/SingleFile#981] [General]
SinleFile update 15/06/22 version 1.21.15 stops after 1/3 with this URL  https://www.hln.be/mijn-gids/expert-legt-uit-hoe-je-asbest-herkent-in-woning-je-vindt-het-in-bouwmaterialen-waarvan-je-het-nooit-zou-vermoeden~a3309aafc/    is the URL too long or ...
Greetings
Henri

**Comment by gildas-lormeau** [comment]
Hi Henri,

I cannot reproduce this issue. Did you change any setting? What browser/version are you using?

**Comment by HenriRT** [comment]
Hello Gildas,

The system is 64 bits with X64 processor. Windows 11 Home 21H2 and 
Firefox Browser 101.0.1 (64bit)The error is only with this url the 
others do. Too many photos or…

https://www.hln.be/mijn-gids/expert-legt-uit-hoe-je-asbest-herkent-in-woning-je-vindt-het-in-bouwmaterialen-waarvan-je-het-nooit-zou-vermoeden~a3309aafc/

No settings have been changed by me

Greetings

Henri



Op 17/06/2022 om 20:43 schreef Gildas:

**Comment by gildas-lormeau** [comment]
Thank you for the additional info. I did a new test with ad-blockers disabled but I still cannot reproduce the issue. Do you have other extensions installed?
